### ISSUE TITLE
Giga Chad Token not craftable

### ISSUE THREAD

**Issue by Usernm0** [GTNewHorizons/GT-New-Horizons-Modpack#12224]
### Your GTNH Discord Username

Usernm#4396

### Your Pack Version

GTNH 2.2.8

### Your Server

Private

### Type of Server

Ultramine

### Your Expectation

Try to craft Giga Chad Token in DTPF. Recipe ask 100m catalyst. But we can't craft MAX input hatch. We can craft only UMV input hatch. We try to use 4 UMV input hatches. DTPF accepts this setup, but recipe not starts after we fill them. 

We take MAX input hatch from creative and fill it with catalyst. Recipe starts.

### The Reality

Recipe not starts :(

### Your Proposal

One of these:

Fix DTPF to take one fluid from multiple hatches for craft.

Make MAX input hatch craftable. 

Make giant input hatch version. We already have giant output hatches. 

Change recipe to fit one UMV input hatch (32m litres)

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by chochem** [comment]
have you tried if the liquid air input hatch works? GDC (who made such a token) suggested it might.

**Comment by YTXilo** [comment]
yeah and the only way to input non fluid into that input hatch is manual

**Comment by chochem** [comment]
ah well, with that method being fixed, the recipe needs a proper fix for sure. I guess it is up to @Connor-Colenso if reducing the fluid amount is ok or more changes are needed.
